README.md

```diff
@@ -60,7 +60,7 @@ _If_ you are using angular-cli the css can be added to your angular-cli.json
 ### Work flow
 
 - Clone repository to your machine.
-- Run `npm run setup` to prepare the project
+- Run `npm install --force` in both root and demo directories.
```

Comment:
Yeah, changed the command because the setup script appears both broken and unnecessary.  We can remove it from package.json.  If the "run demo" command calls for npm install then we can specify npm install in the root only.  The --force flag is to override any issues determining dependency trees.  If you have a cleaner solution for that let me know.  This was only my first foray in to the land of npm.